src/sql/src/plan/statement/ddl.rs

```diff
@@ -2778,6 +2755,8 @@ pub fn plan_create_cluster(
                 size,
                 availability_zones,
                 compute,
+                // Managed clusters do not use `disk_cluster_replicas_default`.
```

Comment:
That doesn't seem right. Why shouldn't they?

Reply:
cc @guswynn

Reply:
updated! managed clusters now use `disk_cluster_replicas_default`